src/resolvers/dns.js

```diff
@@ -3,7 +3,7 @@
 let dns
 
 try {
-  dns = require('dns').promises
+  dns = require('dns').Resolver
```

Comment:
Why this change? This should be using the promises API https://nodejs.org/dist/latest-v14.x/docs/api/dns.html#dns_dns_promises_api